pycue/opencue/api.py

```diff
@@ -260,10 +260,10 @@ def findJob(name):
 
 @util.grpcExceptionParser
 def getJob(uniq):
-    """Returns a Job object for the given job name.
+    """Returns a Job object for the given job id.
     This will only return one or zero active job.
-    @type  name: str
-    @param name: A job name
+    @type  uniq: a unique identifier.
+    @param uniq: id
```

Comment:
I think that the `@type` should be `str`, and `@param` should be "a unique identifier", right?

Reply:
Thanks, I think you're probably correct (i.e. the way you explain it makes more sense to me), but the existing code seems to consistently explain @type and @param in as I have it in the PR. For example:

```
@util.grpcExceptionParser
def getHost(uniq):
    """Returns a Host object from a unique identifier
    @type  uniq: a unique identifier.
    @param uniq: an id
...
```

See also getLayer().

Any idea why the original docstrings might be written this way? (I had a look at the Java src and proto files but that didn't help.) 

There's a lot of work to do on the docstrings, but I was hoping to focus initially on PyOutline rather than PyCue, as I suspect that outline is more generally useful to our user base?

If you're not sure, then I can just swap all of the @type and @param instances for `uniq` in this file and we can leave it at that for now until we clean this up properly.

Reply:
The `type` docstring should reference the object type of the parameter. I feel anything that doesn't is incorrect and will lead to confusion. We should either do as you say and replace all incorrect instances or at least fix this one and create a separate issue to cleanup the docstrings thoughout the whole code base.

Reply:
Yeah, agreed. Totally fine to do the larger cleanup in a separate issue.

Reply:
Thanks both, I'll take another look at this next week as I'm traveling today.

Reply:
I've had a go at cleaning up this file a bit. It's not finished, but there are somethings I'm not really clear about yet, especially `getHostWhiteboard()` and operations related Allocations.

For now, I think it's in a reasonable place and my aim is to leave it so that everything in there is accurate, even if there's some missing info in the short term.